src/TGameDetails.h

```diff
@@ -64,6 +64,28 @@ class TGameDetails
     // clang-format off
     // games are to be added here in alphabetical order
     inline static const QList<GameDetail> scmDefaultGames = {
+            { qsl("Abandoned Realms"),
+            qsl("abandonedrealms.com"),
+            9000,
+            false,
+qsl("<a href='https://abandonedrealms.com'>Website</a><br>"
+                "<a href='https://abandonedrealms.com/forum'>Forum</a>"),
+            qsl(":/icons/abandonedrealms_icon.jpg"),
+            qsl("The Abandoned Realms is a completely free RP-enforced, PK-encouraged, MUD in a fantasy setting "
+                 "(swords/magic/lore/intrigue) that has been up and running constantly since 1997.\n\nWith 20 races and "
+                 "16 classes, there are vast possibilities for players to explore and enjoy. The Abandoned Realms has a "
+                 "dedicated staff of highly skilled immortals/coders/builders, and a diverse base players online at any "
+                 "given time. There are extensive opportunities for role-play, including writing character background "
+                 "entries, building your own house (including decor), contributing to the in-game newspaper, engaging in "
+                 "political intrigue, choosing a deity to worship (and opening exceptional Immortal RP opportunities), and "
+                 "more.\n\nIt is a dangerous realm, filled with exceptional killers but new players are not hunted – they "
+                 "are welcomed and supported. Perma-death exists, but it is common for characters to live for hundreds of "
+                 "hours, or as long as the player desires. Information and advice are shared freely – see The Codex for item "
+                 "identifications/locations, the logboard for battles, and the forum and discord for gameplay discussion and "
+                 "advice.\n\nIf you are interested in a completely free, immersive RP/PK experience, in a vibrant land filled "
+                 "with lore/intrigue, where you can forge long-lasting friendships and bitter rivalries, and have an "
+                 "opportunity to leave your lasting mark and be remembered forever, then welcome ye to the Abandoned Realms.")
```

Comment:
With other entries I think I separated out the double line-feeds so as to pick out each paragraph (and confined the source text - which gets wrapped on display - to 80 character from the first printable one in each line (even though it is further offset by code indentation)...
```suggestion
            qsl("The Abandoned Realms is a completely free RP-enforced, PK-encouraged, MUD in a fantasy setting "
                 "(swords/magic/lore/intrigue) that has been up and running constantly since 1997."
                 "\n\n"
                 "With 20 races and 16 classes, there are vast possibilities for players to explore and enjoy. "
                 "The Abandoned Realms has a dedicated staff of highly skilled immortals/coders/builders, "
                 "and a diverse base players online at any given time. There are extensive opportunities for "
                 "role-play, including writing character background entries, building your own house "
                 "(including decor), contributing to the in-game newspaper, engaging in political intrigue, "
                 "choosing a deity to worship (and opening exceptional Immortal RP opportunities), and "
                 "more."
                 "\n\n"
                 "It is a dangerous realm, filled with exceptional killers but new players are not hunted – they "
                 "are welcomed and supported. Perma-death exists, but it is common for characters to live "
                 "for hundreds of  hours, or as long as the player desires. Information and advice are shared "
                 "freely – see The Codex for item identifications/locations, the logboard for battles, and the "
                 "forum and discord for gameplay discussion and advice."
                 "\n\n"
                 "If you are interested in a completely free, immersive RP/PK experience, in a vibrant land filled "
                 "with lore/intrigue, where you can forge long-lasting friendships and bitter rivalries, and have an "
                 "opportunity to leave your lasting mark and be remembered forever, then welcome ye to the "
                 "Abandoned Realms.")
```